strapi/strapi
# fix: Content-Type-Builder removes hidden fields on save

### What does it do?

Restore non-visible attrs from previous schema before saving content type

### Why is it needed?

`content-type` api returns only visible fields so Content-Type-Builder shows and sends only visible fields on updating, this process removes non-visible fields.

### How to test it?

- Create a content type with a field configuration set to `"visible": false`
- Use CTB to make some changes
- Save changes
- Check schema.json file to make sure all `"visible": false` attributes are kept

### Related issue(s)/PR(s)

Closes https://github.com/strapi/strapi/issues/16283
